.github/workflows/generate.yml

```diff
@@ -0,0 +1,109 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Generate Library & Open Pull Request
+
+# Uses protos from HEAD of https://github.com/googleapis/google-cloudevents
+# and tooling from HEAD of this repository to update all generated code.
+#
+# Creates a PR whose branch will include an unmodified snapshot of the
+# tooling used to generate changes.
+
+on:
+  schedule:
+    # Run at the end of every day.
+    - cron: 0 0 * * *
```

Comment:
maybe choose a different time as midnight feels like it would probably be overloaded. maybe 10pm instead? so it's at the end of the day?

Reply:
Fair, I copied in the previous value.

How about 10:10pm in case other systems kick off on the hour? Making it happen.